openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -265,6 +265,9 @@ message DeviceExt {
   // IDFV of the device in that publisher. Listed as ifv to match ifa field
   // format.
   optional string ifv = 2;
+
+  // Third party cookies deprecation label. The label as received from Chrome or an upstream partner
```

Comment:
// Chrome-facilitated cookie deprecation testing label as received from
  // Chrome.
  // https://developer.chrome.com/en/docs/privacy-sandbox/chrome-testing/
as described in https://developers.google.com/authorized-buyers/rtb/downloads/open-beta/openrtb-adx-proto